pkg/cri/server/container_create_linux_test.go

```diff
@@ -1647,8 +1650,12 @@ containerEdits:
 			}
 			require.NoError(t, err)
 
-			injectFun := oci.WithCDI(test.annotations, []string{cdiDir})
-			err = injectFun(nil, nil, nil, spec)
+			reg := cdi.GetRegistry()
+			err = reg.Configure(cdi.WithSpecDirs(cdiDir))
```

Comment:
Just thinking that the `GetRegistry` returns global vars. So, does the last `Configure` call impact the next call?

Reply:
It does as the registry is a singleton: https://github.com/container-orchestrated-devices/container-device-interface/blob/main/pkg/cdi/registry.go#L127
So, next call of GetRegistry returns the same registry.

Reply:
the Configure function will update the registry, right? How to make sure that the next test case won't be impacted by the last one?

Reply:
sorry for misunderstanding. the registry is reconfigured for each test case, which means it's fully initialized with the new cdiDir. It would impact the next test case if it's not reconfigured.